synq/api2.go

```diff
@@ -103,6 +107,28 @@ func (a *ApiV2) Create(body ...[]byte) (VideoV2, error) {
 	return video, nil
 }
 
+func (a *ApiV2) GetVideos(accountId string) (videos []VideoV2, err error) {
+	var resp VideoList
+	path := "/videos"
+	if accountId != "" {
+		path = "/" + accountId + path
```

Comment:
I think this is producing `/:account_id/videos` not `/accounts/:account_id/videos`

Reply:
oops, haha